deploy/deployqt.cmake

```diff
@@ -7,9 +7,9 @@ elseif(WIN32)
     set(DEPLOYQT_NAME win)
     if(${QT_VERSION_MAJOR} GREATER_EQUAL 6)
         list(JOIN LANGS "," QT_LANGS)
-        set(DEPLOYQT_ARGS --no-quick-import --translations ${QT_LANGS})
+        set(DEPLOYQT_ARGS --no-quick-import -xml -qml  -core5compat --translations ${QT_LANGS})
```

Comment:
two spaces

Reply:
its only one for -<module> . oddly inconstant for Qt.